ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -224,6 +227,17 @@ export const CoinOverview = ({
   );
   const isRewardsEnabled = useRewardsEnabled();
 
+  const accountGroupBalance = useSelector(
+    selectAccountGroupBalanceForEmptyState,
+  );
+  const isTestnet = useSelector(getMultichainIsTestnet);
+
+  const shouldShowBalanceEmptyState =
+    isMultichainAccountsState2Enabled &&
+    !isTestnet &&
+    !balanceIsCached &&
+    accountGroupBalance === 0;
```

Comment:
Shows empty state only when ALL conditions are met:
1. Multichain accounts state 2 is enabled (new account system)
2. Not on a testnet (testnets always show balance, even if 0)
3. Balance data is fresh (not cached/stale)
4. Aggregated balance across ALL mainnet networks is exactly 0